src/components/Card/Card.tsx

```diff
@@ -49,15 +46,15 @@ class Card extends React.Component<Props, State> {
 
   render () {
     const { windowWidth } = this.state
-    const { onPress, factor = 1, icon, title, imageUrl, minorText: minor, className, style } = this.props
+    const { onPress, factor = 1, icon, imageUrl, className, style } = this.props
     const width = this._self ? this._self.clientWidth : windowWidth / factor
+    var {title} = this.props
```

Comment:
You can move this to line 49

Reply:
Can't it not be a const because I'm going to potentially truncate it?

Reply:
Alright; at least change it to a `let`, otherwise the linter will complain